alpha/alpha-server/src/main/java/org/apache/servicecomb/pack/alpha/server/fsm/GrpcSagaEventService.java

```diff
@@ -54,11 +54,32 @@ public GrpcSagaEventService(ActorEventChannel actorEventChannel,
   }
 
   @Override
-  public void onConnected(
-      GrpcServiceConfig request, StreamObserver<GrpcCompensateCommand> responseObserver) {
-    omegaCallbacks
-        .computeIfAbsent(request.getServiceName(), key -> new ConcurrentHashMap<>())
-        .put(request.getInstanceId(), new GrpcOmegaCallback(responseObserver));
+  public StreamObserver<GrpcServiceConfig> onConnected(StreamObserver<GrpcCompensateCommand> responseObserver) {
+    return new StreamObserver<GrpcServiceConfig>(){
+
+      private GrpcOmegaCallback grpcOmegaCallback;
+
+      @Override
+      public void onNext(GrpcServiceConfig grpcServiceConfig) {
+        grpcOmegaCallback = new GrpcOmegaCallback(responseObserver);
+        omegaCallbacks
+            .computeIfAbsent(grpcServiceConfig.getServiceName(), key -> new ConcurrentHashMap<>())
+            .put(grpcServiceConfig.getInstanceId(), grpcOmegaCallback);
+        responseObserver.onNext(GrpcCompensateCommand.newBuilder()
+            .setConnectedResponse(true)
+            .build());
```

Comment:
@WillemJiang Alpha sends a response message to Omega after receiving the connection request.